core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/TSqlExpressionBuilder.scala

```diff
@@ -392,6 +392,11 @@ class TSqlExpressionBuilder() extends TSqlParserBaseVisitor[ir.Expression] with
     ctx.expression().accept(this)
   }
 
+  override def visitPartitionFunction(ctx: PartitionFunctionContext): ir.Expression = {
+    // $$PARTITION is not supported in Databricks SQL, so we will report it is not supported
+    functionBuilder.buildFunction(s"$$PARTITION", List.empty)
```

Comment:
```suggestion
    functionBuilder.buildFunction("$PARTITION", List.empty)
```

Reply:
I cannot do this as the Scala compiler incorrectly complains that "$PARTITION" is a an unresolved interpolation. It is apparently a known issue and I had to do quite a bit of searching to work out why it was saying that. It only seems to happen when the string is the first parameter of a function.